examples/entity-aggregation/src/main/java/org/example/ImportCustomers.java

```diff
@@ -0,0 +1,70 @@
+package org.example;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.spark.sql.SaveMode;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.functions;
+
+import java.util.Properties;
+
+public class ImportCustomers {
+
+    public static void main(String[] args) {
+        if (args.length == 0 || StringUtils.isEmpty(args[0].trim())) {
+            System.err.println("Please ensure mlPassword is set before running. See the README.md for more information.");
+            return;
+        }
+        final String markLogicAdminPassword = args[0];
+
+        // Create a vanilla local Spark session.
+        SparkSession session = SparkSession.builder()
+            .master("local[*]")
+            .config("spark.sql.session.timeZone", "UTC")
+            .getOrCreate();
+
+        // Define the JDBC properties for connecting to Postgres.
+        Properties jdbcProperties = new Properties();
+        jdbcProperties.setProperty("driver", "org.postgresql.Driver");
+        jdbcProperties.setProperty("user", "postgres");
+        jdbcProperties.setProperty("password", "postgres");
+
+        // See https://stackoverflow.com/a/39129546 for information on aliased queries with Spark. A table name may be
+        // used, but in this example, we want to join customers and rentals together. The query is placed in parentheses
+        // with an alias after it; the alias can have any name.
+        String query = "(select c.*, r.rental_id, r.rental_date, r.return_date " +
+            "from customer c " +
+            "inner join public.rental r on c.customer_id = r.customer_id " +
+            ") the_query ";
+
+        session
+            .read()
+            .jdbc("jdbc:postgresql://localhost/dvdrental", query, jdbcProperties)
+            // Group all the rows by customer_id.
+            .groupBy("customer_id")
```

Comment:
The following agg function call is complex and it's where the important magic is happening.
I wonder if there should be something in here pointing to the example JSON in the README file and specifically telling users that this code is transforming multiple rows into single rows that (eventually) look like the JSON there.